src/tests/CMakeLists.txt

```diff
@@ -20,10 +20,10 @@ endif()
 
 if(IDYNTREE_USES_ICUB_MAIN)
     # Add benchmark test against yarp/icub functions
-    add_subdirectory(yarp_benchmark)
+    #add_subdirectory(yarp_benchmark)
 
     # Add a subdirectory of all the consistency tests between the old
     # iCub-dependent external force/internal torque estimation
     #  implementations and the new native iDynTree implementations
-    add_subdirectory(icub_consistency)
+    #add_subdirectory(icub_consistency)
```

Comment:
We should revert this changes are rebase this branch on the top of the `devel` branch.

Reply:
Uncommented.